src/context/signInContext.js

```diff
@@ -47,8 +51,9 @@ export function SignInContextProvider({children}) {
      */
     const [currentUsername, setCurrentUsername] = useLocalStorage('currentUsername', '');
     const [currentPassword, setCurrentPassword] = useLocalStorage('currentPassword', '');
+    const [currentUserId, setCurrentUserId] = useLocalStorage('currentUserId', '');
     const [cisBasePath] = useLocalStorage('cisBasePath', CIS_QA_URL);
-    const attemptSignIn = (username, password) => signInWithCredentials(cisBasePath, username, password, signInInProgress, setSignInInProgress, setSignedIn, setCurrentUsername, setCurrentPassword);
+    const attemptSignIn = (username, password) => signInWithCredentials(cisBasePath, username, password, signInInProgress, setSignInInProgress, setSignedIn, setCurrentUsername, setCurrentPassword, setCurrentUserId);
     const attemptSignup = (username, email, password) => signUpWithCredentials(cisBasePath, username, email, password, setSignupState);
     /**
      * To be used
```

Comment:
```suggestion
     * To be used when we encounter a `401` which would indicate our credentials are invalid. Resetting all this info 
     * will effectively boot the user back to the login screen where they will have to sign in again.
```
Oops. Look like I forgot to write this whenever I added this comment. Feel free to accept this suggestion.